apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/RODViewPoolCandidatePage.tsx

```diff
@@ -646,20 +646,10 @@ export const ViewPoolCandidate = ({
                 })}
               </p>
               <p>Step 1: Screening Application (Replace with fetched value)</p>
```

Comment:
Is **Replace with fetched value** meant to remain?

## Screenshot
<img width="304" alt="Screen Shot 2024-02-12 at 09 39 13" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/762b0668-2197-4339-9be8-c23cd17c42d5">

Reply:
My guess is that it will be similar to [the pill we recently added](https://github.com/GCTC-NTGC/gc-digital-talent/issues/9146) but include the step name rather than the status 🤔 

@Jerryescandon is that accurate?

Reply:
Ok, will this be removed as part of another PR then? I just want to make sure this is out of scope for this issue.